active/NavigationView/TopPadding/TopPaddingMargins.md

```diff
@@ -0,0 +1,72 @@
+> See comments in Markdown for how to use this spec template
```

Comment:
This line is showing up in your rich diff.

Reply:
@StephenLPeters - seeing this line show up in the doc confused me, so I deleted it along with all the comments. This line isn't part of the spec text.